src/api/config/routes.rb

```diff
@@ -637,7 +637,7 @@ def self.public_or_about_path?(request)
       get 'public/source/:project/:package/:filename' => :source_file, constraints: cons
       get 'public/distributions' => :distributions
       get 'public/binary_packages/:project/:package' => :binary_packages, constraints: cons
-      get 'public/build/:project(/:repository(/:arch(/:package(/:file))))' => 'public#build', constraints: cons, as: :public_build
+      get 'public/build/:project(/:repository(/:arch(/:package(/:filename))))' => 'public#build', constraints: cons, as: :public_build
```

Comment:
Ah, nevermind. file already got replaced in the controller [4 years](54a74c760ab8e313f64b6d6d4efd0ea5ab047cdb) ago.